assignmentPR/MathFunc1.c

```diff
@@ -0,0 +1,316 @@
+/* 
+
+MathFunc01.c
+Written by Stephen White, Kalen Wood-Wardlow, & Gary Baker
+9/23/18
+
+*/
+
+/* Import Statements (no math allowed) */
+#include <stdlib.h>
+#include <stdio.h>
+
+
+/* Global Constants */
+typedef int bool;
+#define true 1
+#define false 0
+
+const int CHAR_ARRAY_SIZE_MAX = 80; 
+const int INT_CONVERSION_ERROR = -1;
+const int WHITESPACE = 32; 
+const int SAME_STRING = 0;
+
+
+/* Function Declarations */
+
+bool characteristic(char numString[], int *c);
+
+bool mantissa(char numString[], int *numerator, int *denominator);
+
+int compareString(const char *stringOne, const char *stringTwo);
+
+void cleanCharArray(char *charArray, int arraySize);
+
+int isNumericChar(char character);
+
+int myAtoi(char *stringSequence);
+
+void reverse(char string[]);
+
+int myStrLen(char *string);
+
+int tenToThePowerOf(int stringLength);
+
+
+/* Function implementations */
+
+int main(int argv,  char *argc)
```

Comment:
Flipped var names from the normal declaration of main func. I know these parameters aren't used, but just thought it could overlooked if they were used.

Usually:
`int main(int argc, char *argv[])`

Reply:
Thanks I will fix that right now!